### PR TITLE
Fix client side files position

### PR DESCRIPTION
**Pull request by SamueleFurnari**
There is a little error in the sample: if you follow the tutorial, the `npm run release` fails because it looks for html/css/ts files in the src folder but the tutorial says to create these files in the project root.

My favourite solution is to create the _src_ folder and then put these files into this folder. I modified the tutorial to follow this solution.

Thanks!

Samuele

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/signalr-typescript-webpack.md](https://github.com/dotnet/AspNetCore.Docs/blob/7adf0ef7219ac7d78e49d5a9d553a119a9fb3827/aspnetcore/tutorials/signalr-typescript-webpack.md) | [Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Webpack](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/signalr-typescript-webpack?branch=pr-en-us-30173) |

### CODE REVIEW

**Comment by guardrex** [comment]
Thanks, @SamueleFurnari.

@ssougnez ... I see you're listed as the author. Would you like to take a look?

**Comment by guardrex** [comment]
*Stand-by* ... I emailed Sebastien to let him know about this PR.

**Comment by ssougnez** [comment]
On the other side, you only left me one day to answer which is pretty short...

I'll try to have a look at this in the upcoming days but i cannot promise anything (lot of work nowadays)

**Comment by Rick-Anderson** [comment]
@ssougnez let me know if you'll have time soon to review this PR. No hurry.

**Comment by Rick-Anderson** [comment]
@mkArtakMSFT please assign a reviewer

**Comment by Rick-Anderson** [comment]
@wadepickett  can you test this?

**Comment by wadepickett** [comment]
Sure, I can test it Monday.  Sorry I missed this Friday end of day.  I was helping a PM at the time and didn't notice it.

File: aspnetcore/tutorials/signalr-typescript-webpack.md
Comment (wadepickett) [inline aspnetcore/tutorials/signalr-typescript-webpack.md:178]:
```suggestion
1. Copy the `src` directory and its contents from the [sample project](https://github.com/dotnet/AspNetCore.Docs.Samples/tree/main/tutorials/signalr-typescript-webpack/samples/8.x/SignalRWebpack/) into the project root. The `src` directory contains the following files:
```

Comment (wadepickett) [reply aspnetcore/tutorials/signalr-typescript-webpack.md:178]: The issue here I think is that the link was to the content of the `src`, and should have been a level up, so that it was clear you are to copy the `src` directory to the root as instructed.  By linking to `src` itself, you could accidentally think you are to copy the contents of the directory only, rather than the directory `src` and its contents.  I clarified "and its contents" and fixed the link.  Thanks for bringing this up, it wasn't as clear as it could be.

**Review by wadepickett — CHANGES_REQUESTED** [review]
@Rick-Anderson, It runs fine, but I made some changes to both clarify and reduce the instruction needed.  Since you took the assignment, I will leave it to you to approve/merge if that seems right.

**Comment by wadepickett** [comment]
Sorry, Rick, I just realized the changes also needed to be made to version 7 and 6.  I should have let you know I was changing them.  :(  I put them in the same time you merged.  Looks like auto-merge might still take them.  watching...

**Comment by wadepickett** [comment]
All in, verified.  Thanks.  Done.

**Pull request merged**
